src/ansys/fluent/core/services/field_data.py

```diff
@@ -638,7 +638,56 @@ class PathlinesFieldDataRequest(NamedTuple):
     zones: list | None = None
 
 
-class TFieldData:
+class _IFieldData:
+    """The base field data interface."""
```

Comment:
@prmukherj No actual interface is being enforced here. If we want to follow such an approach, we need to go all in or not at all. There is a standard approach to do it in Python.